_data/master/navbars/security.yml

```diff
@@ -4,103 +4,71 @@ order: 3
 toc:
 - title: Security
   path: /security/
-
```

Comment:
@bmckercher123 could you undo these whitespace changes? They cause merge pain in the future. We shouldn't make unnecessary cosmetic changes if it can be avoided.

Reply:
DONE